### PR TITLE
Add in a test for MR04

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/support/page_objects/SatisfyMortgageChargePage.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        cy.get('#Name').type(name);
+        return this;
+    }
+
+    clickEnterAddressManually() {
+        cy.get('#residential-address-manual-link').click();
+        return this;
+    }
+
+    enterHomeAddress(propertyNumber, postcode) {
```
Comment (rsuller) [inline cypress/support/page_objects/SatisfyMortgageChargePage.js]: Is this address sort of stuff starting to become duplicated?
I remember you mentioning something earlier about it. I did create an `AddressPage` class in the `generic` package. Is this potentially something we should look to refactor?

Comment (danfalc) [reply cypress/support/page_objects/SatisfyMortgageChargePage.js]: Yeah it definitely is. I'll do it as part of a separate refactoring PR from this one

Comment (danfalc) [reply cypress/support/page_objects/SatisfyMortgageChargePage.js]: If that is acceptable?

Comment (rsuller) [reply cypress/support/page_objects/SatisfyMortgageChargePage.js]: 👍

**Pull request merged**